### PR TITLE
Expose ARN for service so we don't have to manually construct it

### PR DESCRIPTION
**Pull request by X82Softworks**
## what
* Expose ARN for service so we don't have to manually construct it

## why
* Ease of use and preventing user error

### CODE REVIEW

**Review by jamengual — DISMISSED** [review]

**Review by nitrocode — DISMISSED** [review]

**Comment by X82Softworks** [comment]
Please let me know what changes I can make on this side to allow access and fix the 403s you're getting from your automated tools

**Comment by SweetOps** [comment]
@X82Softworks take a look on error:

```sh
--- FAIL: TestExamplesComplete (19.82s)
442
    apply.go:15: 
443
        	Error Trace:	apply.go:15
444
        	            				examples_complete_test.go:26
445
        	Error:      	Received unexpected error:
446
        	            	FatalError{Underlying: error while running command: exit status 1; 
447
        	            	Error: Unsupported attribute
448
        	            	
449
        	            	  on ../../outputs.tf line 18, in output "service_arn":
450
        	            	  18:   value       = join("", aws_ecs_service.default.*.arn) != "" ? join("", aws_ecs_service.default.*.arn) : join("", aws_ecs_service.ignore_changes_task_definition.*.arn)
451
        	            	
452
        	            	This object does not have an attribute named "arn".
453
        	            	}
454
        	Test:       	TestExamplesComplete
455
FAIL
456
exit status 1
```

File: outputs.tf [hunk 1]
```diff
@@ -13,6 +13,11 @@ output "service_name" {
   value       = join("", aws_ecs_service.default.*.name) != "" ? join("", aws_ecs_service.default.*.name) : join("", aws_ecs_service.ignore_changes_task_definition.*.name)
 }
 
+output "service_arn" {
+  description = "ECS Service ARN"
+  value       = join("", aws_ecs_service.default.*.arn) != "" ? join("", aws_ecs_service.default.*.arn) : join("", aws_ecs_service.ignore_changes_task_definition.*.arn)
```
Comment (SweetOps) [inline outputs.tf:18]:
```suggestion
  value       = join("", aws_ecs_service.default.*.id) != "" ? join("", aws_ecs_service.default.*.id) : join("", aws_ecs_service.ignore_changes_task_definition.*.id)
```

**Review by SweetOps — CHANGES_REQUESTED** [review]

**Comment by SweetOps** [comment]
@X82Softworks could you please allow upstream maintainers to make changes to your branch?
https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork#:~:text=to%20your%20branch.-,Enabling%20repository%20maintainer%20permissions%20on%20existing%20pull%20requests,like%20to%20allow%20commits%20on.

**Comment by X82Softworks** [comment]
@SweetOps There's no option available on my PR to allow that, do I need to have elevated permissions elsewhere to enable it?

**Comment by SweetOps** [comment]
@X82Softworks , I don't know... in my PRs that checkbox is present ...

run the following commands locally and commit:

```
make init
make readme
terraform fmt
```

**Review by SweetOps — APPROVED** [review]

**Pull request merged**